src/main/java/io/prismic/Api.java

```diff
@@ -297,19 +297,33 @@ public Ref getMaster() {
   }
 
   /**
+   * Start a query defaulting on the master reference (you can still override it) with the given language
+   */
+  public Form.SearchForm query(String lang, String q) {
```

Comment:
Yes, you're right! That would also make the API nicer (`lang` in front of the `query` param was weird anyway). Will remove.